<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qtClass</class>
 <widget class="QMainWindow" name="qtClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1172</width>
    <height>901</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qt</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QToolButton" name="PC1_toolButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>91</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>PC1</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Resources/PC-light.png</normaloff>Resources/PC-light.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="PC2_toolButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>340</y>
      <width>91</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>PC2</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Resources/PC-light.png</normaloff>Resources/PC-light.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="PC3_toolButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>40</y>
      <width>91</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>PC3</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Resources/PC-light.png</normaloff>Resources/PC-light.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="PC4_toolButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>340</y>
      <width>91</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>PC4</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Resources/PC-light.png</normaloff>Resources/PC-light.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="PC5_toolButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>40</y>
      <width>91</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>PC5</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Resources/PC-light.png</normaloff>Resources/PC-light.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="PC6_toolButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>340</y>
      <width>91</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>PC6</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Resources/PC-light.png</normaloff>Resources/PC-light.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>611</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="TB1_toolButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>200</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>网桥1</string>
    </property>
    <property name="icon">
     <iconset resource="qt.qrc">
      <normaloff>:/qt/Resources/TB.png</normaloff>:/qt/Resources/TB.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="TB2_toolButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>200</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>网桥2</string>
    </property>
    <property name="icon">
     <iconset resource="qt.qrc">
      <normaloff>:/qt/Resources/TB.png</normaloff>:/qt/Resources/TB.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QTableView" name="PC_detailTable">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>350</y>
      <width>381</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="TB1_detailTable">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>480</y>
      <width>471</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>270</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QToolButton" name="goAhead_toolButton">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>270</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>快进(秒)</string>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextOnly</enum>
    </property>
    <property name="autoRaise">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTableView" name="TB2_detailTable">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>480</y>
      <width>471</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="message_lineEdit">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>180</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="send_comboBox">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>60</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>PC1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>PC2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>PC3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>PC4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>PC5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>PC6</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>40</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>发送方</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>90</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>接收方</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>160</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>消息内容</string>
    </property>
   </widget>
   <widget class="QToolButton" name="sendMessage_toolButton">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>150</y>
      <width>61</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
    <property name="icon">
     <iconset resource="qt.qrc">
      <normaloff>:/qt/Resources/ok.jfif</normaloff>:/qt/Resources/ok.jfif</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="recv_comboBox">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>110</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="currentText">
     <string>PC6</string>
    </property>
    <property name="currentIndex">
     <number>5</number>
    </property>
    <item>
     <property name="text">
      <string>PC1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>PC2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>PC3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>PC4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>PC5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>PC6</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>460</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>网桥1转发表</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>460</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>网桥2转发表</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>320</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>PC信息</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>90</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>接收方MAC(可重新编辑)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>40</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>发送方MAC(可重新编辑)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="senderMAC_lineEdit">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>60</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="recverMAC_lineEdit">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>110</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QTimeEdit" name="timeEdit">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>270</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="displayFormat">
     <string>HH:mm:ss</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>240</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>当前时刻</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="messageBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>690</y>
      <width>1021</width>
      <height>141</height>
     </rect>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>670</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>输出</string>
    </property>
   </widget>
   <zorder>graphicsView</zorder>
   <zorder>PC5_toolButton</zorder>
   <zorder>PC3_toolButton</zorder>
   <zorder>PC1_toolButton</zorder>
   <zorder>PC4_toolButton</zorder>
   <zorder>PC6_toolButton</zorder>
   <zorder>PC2_toolButton</zorder>
   <zorder>TB1_toolButton</zorder>
   <zorder>TB2_toolButton</zorder>
   <zorder>PC_detailTable</zorder>
   <zorder>TB1_detailTable</zorder>
   <zorder>spinBox</zorder>
   <zorder>goAhead_toolButton</zorder>
   <zorder>TB2_detailTable</zorder>
   <zorder>message_lineEdit</zorder>
   <zorder>send_comboBox</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>sendMessage_toolButton</zorder>
   <zorder>recv_comboBox</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>label_8</zorder>
   <zorder>senderMAC_lineEdit</zorder>
   <zorder>recverMAC_lineEdit</zorder>
   <zorder>timeEdit</zorder>
   <zorder>label_9</zorder>
   <zorder>messageBox</zorder>
   <zorder>label_10</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1172</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qt.qrc"/>
 </resources>
 <connections/>
</ui>
